src/components/tabs/Tabs.module.css

```diff
@@ -0,0 +1,31 @@
+.container {
+	padding-block: 1rem;
+}
+
+.tablist {
+	display: flex;
+	overflow-x: scroll;
+	/* Hide scrollbar on firefox */
+	scrollbar-width: none; 
+}
+
+.tablist::-webkit-scrollbar {
+	/* Hide scrollbar on chrome / safari */
+  display: none;
+}
+
+.tab {
+	border-radius: unset;
+	background-color: unset;
+	padding: 0.5rem 1rem;
+	border-bottom: 4px solid var(--theme-divider);
+	cursor: pointer;
+}
+
+.tab[aria-selected=true] {
+	border-bottom-color: var(--theme-accent);
+}
+
```

Comment:
Looks like focus styling is getting clipped now that the tab list can be scrolled:

<img width="181" alt="image" src="https://user-images.githubusercontent.com/357379/177872611-a68e02e7-f11c-4996-945c-0bf58d3eada9.png">


How about something like this to fix that?

```suggestion

.tab:focus {
	outline: 2px solid var(--theme-accent);
	outline-offset: -2px;
}

```

<img width="183" alt="image" src="https://user-images.githubusercontent.com/357379/177872643-15a96dbe-1bf0-4c1e-9cf4-be139abe6346.png">